rotkehlchen/assets/converters.py

```diff
@@ -555,7 +569,23 @@ def asset_from_bitfinex(
     return Asset(name)
 
 
+def asset_from_bitstamp(bitstamp_name: str) -> Asset:
+    """May raise:
+    - DeserializationError
+    - UnknownAsset
```

Comment:
It's good, but it's mostly useful for subfunctions to know the flow of exception raising.

For example: UnknownAsset due to Asset constructor here.

It's hard to maintain ... not sure if it's a good idea but I have noticed that it has helped me when looking at a piece of codes months after writing it.

Ideally a tool would exist that does it for us. But I don't think one exists